### ISSUE TITLE
2019-10-25

### ISSUE THREAD

**Issue by filonitta** [Anto111a/homework#2]
https://github.com/Anto111a/homework/blob/45f5f27f4c2f2180a61721a28b69a7d0f36dda61/2019-10-25/script.js#L34

:) с тернарным оператором осторожно, код легко может стать нечитаемым. Check it out:
`var result = a > b ? a > 0 ? 'a' : 'b' : b > 0 ? 'b' : 'a'`

https://github.com/Anto111a/homework/blob/45f5f27f4c2f2180a61721a28b69a7d0f36dda61/2019-10-25/script.js#L141
не смутило, что при отмене в переменную `name` записывается строка `"null"`, а не настоящий `null`, как обычно? :)

**Comment by Anto111a** [comment]
@filonitta 
У меня вроде ещё не сложно )
А сложно я наверное и не напишу ещё, потому что сам  запутаюсь пока напишу)

А по поводу name и "null" не обратил внимания когда писал, но немного погуглив нашел, что переменные name всегда преобразуются в строки, хотя для чего это так и не разобрался.

**Comment by shiryaeva-n-dbbest** [comment]
да, использование тернарного оператора для двух вариантов развития событий - допустимо. Просто надо держать в уме, что он легко может стать сложночитаемым

про переменную `name` правда, что при записи в нее все преобразуется в строку. Расскажу, как так на занятии (если не забуду вспомнить)
